resolveConfig.js

```diff
@@ -0,0 +1,6 @@
+const resolveConfigObjects = require('./lib/util/resolveConfig').default
+const defaultConfig = require('./stubs/defaultConfig.stub.js')
+
+module.exports = function resolveConfig(config) {
```

Comment:
@adamwathan Why not just using an arrow function?
```suggestion
module.exports = config => resolveConfigObjects([config, defaultConfig])
```

Reply:
No real good technical reason 🤷‍♂ I like named functions when I don't actually need the features of arrow functions like the `this` binding or closing over the parent scope. Feels more like a tidy isolated little package.

Reply:
Looked a bit cleaner to just have it on one line that's all 😉